src/style.css

```
@@ -0,0 +1,19 @@
+.blocks {
+  margin: 0;
+}
+
+.box {
+  height: 100px;
+}
+
+.redBlock {
```

пиши назви класів через `-`
```suggestion
.red-block {
```

---

исправил